### DISCUSSION TITLE
Create a Website for this Repo

### DISCUSSION THREAD

**Discussion by jfullstackdev** [crev-dev/cargo-crev#444] [Q&A]
Hi there. 

Is there a website for this repo? 

Because if you don't have, well, this repo can simply be turned into a website right away. Others will discover this project in that website.

Steps:
1. Go to Settings and look for `GitHub Pages`, scroll down. That's almost at the bottom.

2. You will see there: `Branch:none`, so you should change that to `master` because you have a `README.md` file in the master repo. This will be your page. Click `Save` first.

3. Then click `Choose a theme`, you select a predefined theme of your site.

4. Visit your site now! The URL will be `https://crev-dev.github.io/cargo-crev`. 

If you were amazed by that, simply read the documentation about GitHub Pages.

**Comment by dpc** [comment]
I own a crev.dev doman, and we do have a web.crev.dev, but I do plan to eventually make http://crev.dev into something informative. I do know GitHub Pages, but haven't considered using it. Maybe I should, until something better is ready. Thanks.

**Comment by jfullstackdev** [comment]
GitHub Pages on its own is actually great already. Thanks for the response.

**Comment by bestia-dev** [comment]
Try the website:
https://web.crev.dev/rust-reviews/crates/
